packages/pn-commons/src/utility/__test__/paymentCaching.utility.test.ts

```diff
@@ -0,0 +1,160 @@
+import { cachedPayments, paymentsData } from '../../__mocks__/NotificationDetail.mock';
```

Comment:
Add two new test cases.
1. invalid object in cache -> null returned
2. invalid json in cache -> null returned